tap_github/streams.py

```diff
@@ -904,3 +904,53 @@ def http_headers(self) -> dict:
             ),
         ),
     ).to_dict()
+
+
+class StargazersStream(GitHubStream):
+    """Defines 'Stargazers' stream. Warning: this stream does NOT track star deletions."""
+
+    name = "stargazers"
+    path = "/repos/{org}/{repo}/stargazers"
+    primary_keys = ["repo", "org"]
+    parent_stream_type = RepositoryStream
+    state_partitioning_keys = ["repo", "org", "user_id"]
```

Comment:
Looks like these should be reversed. The primary key should be including user ID, but the state key (which drives how many bookmarks the stream will track) should be just ["repo", "org"].

Does that sound right?

```suggestion
    primary_keys = ["repo", "org", "user_id"]
    parent_stream_type = RepositoryStream
    state_partitioning_keys = ["repo", "org"]
```